include/glow/Graph/Graph.h

```diff
@@ -222,12 +223,20 @@ class Function final : public Named {
   /// A reference to the owner of the function.
   Module *parent_;
 
+  /// The log context associated with this function.
+  LogContext logCtx_;
+
 public:
   Function(Module *parent, llvm::StringRef Name = {})
-      : Named(Name), parent_(parent) {}
+      : Named(Name), parent_(parent) {
+    logCtx_ = LogContext();
```

Comment:
Why do you need this line?

Reply:
Agreed, this will be constructed automatically.